src/render/tutorials/getting-started-with-docker/2_0.html.md

```diff
@@ -33,120 +34,162 @@ Project has the following structure:
-</body>
-</html>
-```
-
-### Templates repository configuration
-We will use [fsRepoConnectorHandler](https://github.com/Knotx/knotx-repository-connector/tree/master/fs), its purposes are purely academical.
-It is not designed to be used as a production ready solution (you should use [httpRepoConnectorHandler](https://github.com/Knotx/knotx-repository-connector/tree/master/http) there).
-
-Create `knotx/conf/routes/handlers/fsRepoConnectorHandler.conf` file with the following:
-```hocon
-# Path to the directory on the local filesystem which will be the root for requested templates
-catalogue = "/usr/local/knotx/repository/"
-```
-This way we define the `repository` directory that will be our files content repository.
+## Create new routing path
 
-### openapi.yml
-Open `knotx/conf/openapi.yml` and add following path definition:
+Let's take a look at `knotx/conf/openapi.yaml` file.
 
-```
-  /content/*:
+It's a [OpenAPI 3.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md) specification file that defines configuration of our project.
+
+We can see three distinguishable parts of this file:
+
+ 1.  Information about our project, which is quite self-explanatory:
+ 
+      ```
+      ...
+      
+      openapi: "3.0.0"
+      info:
+        version: 1.0.0
+        title: Knot.x Starter Kit
+        description: Knotx Starter Kit
+        
+      ...
+      ```      
+ 2.  Server configuration that defines how our server behaves:
+ 
+      ```
+      ...
+      
+      servers:
+        - url: https://{domain}:{port}
+          description: The local API server
+          variables:
+            domain:
+              default: localhost
+              description: api domain
+            port:
+              enum:
+                - '8092'
+              default: '8092'
+      
+      ...
+      ```
+      Here we can see for example that our server will be registered on port `8092`.
+ 3.  Paths configuration that explicitly specifies our endpoints, their paths, methods, responses and much more in a clear, easily readable and declarative way:
+ 
+      ```
+      ...
+      
+      paths: 
+      
+        ...   
+             
+        /healthcheck:
+          get:
+            operationId: healthcheck-operation
+            # https://vertx.io/docs/vertx-health-check/java/
+            responses:
+              '200':
+                description: Everything is fine
+              '204':
+                description: No procedure are registered.
+              '501':
+                description: At least one procedure has reported a non-healthy stat
+              '500':
+                description: One procedure has thrown an error or has not reported a status in time
+      ...
+      ```
+      Our starter-kit template is comes with some already defined paths out of the box. Let's take the `/healthcheck` above as an example. This endpoint serves purpose of testing whether our dockerized distribution is up and running.
```

Comment:
I would use `Starter Kit` instead of `starter-kit`.

Reply:
done